tests/unit/introspection.tcl

```diff
@@ -61,6 +65,26 @@ start_server {tags {"introspection"}} {
         $rd4 close
     }
 
+tags {"needs:debug"} {
+    test {CLIENT KILL AGE will kill old clients} {
+            set rd1 [redis_deferring_client]
+            r debug sleep 2
+            set rd2 [redis_deferring_client]
+            set connected_clients [s connected_clients]
+            assert {$connected_clients >= 3}
+            # Should kill everything except r and rd2
+            set res [r client kill age 1]
+            assert {$res == $connected_clients - 2}
+
+            # rd2 should still be connected
+            $rd2 ping
+            assert_equal "PONG" [$rd2 read]
```

Comment:
there's an implicit reconnect, so i'm not sure that's a valid test.
maybe we should use `client setname`, and check that it's missing from `client list`?

Reply:
I stole the logic from the `CLIENT KILL SKIPME` test (the one immediately following this one), so I was kinda hoping it was correct.

(It does work, FWIW; the test fails without the new code, and passes with it.)